veniq/baselines/semi/extract_semantic.py

```diff
@@ -48,6 +48,14 @@ def on_node_entering(self, node: Union[Block, Statement]) -> None:
         else:
             raise ValueError(f"Unexpected node type {type(node)}.")
 
+    def on_node_leaving(self, node: Union[Block, Statement]) -> None:
+        if isinstance(node, Block):
+            self._on_block_leaving(node)
+        elif isinstance(node, Statement):
```

Comment:
```
elif not isinstance(node, Statement):
    raise ValueError(f"Unexpected node type {type(node)}.")
```

Looks better, doesn't it?

Reply:
I leaved branch with `pass` for further addition. If we would need to handle `Statements` on node leaving we would easily replace `pass` with needed method.